media/css/mozorg/home/ijustine-promo.scss

```diff
@@ -9,11 +9,6 @@
 
     .mzp-c-split-bg {
         background-color: #6f4ae7;
-        padding-top: 0;
-    }
-
-    .mzp-c-split-container {
-        padding: 0;
```

Comment:
**Nit(blocking):** I declared these values because design showed the iJustine image taking up full width of the container by going all the way to the edges:
![Screen Shot 2023-02-27 at 11 42 24 AM](https://user-images.githubusercontent.com/42309026/221447946-fa1a348d-872d-4cec-ba74-627fc3e847d3.png)


They also wanted the Firefox logo to be removed in mobile. Would you be able to remove this change + remove the Firefox logo in the mobile viewports?

Reply:
(makes zero sense to me too)

Reply:
Going to ask for an explanation in our studio-web chat. Let's see if they agree to place it back in mobile.

Reply:
Seems it was an oversight from studio. Let's bring back the logo 🏋🏻‍♀️